db/scripts/sanitise.sql

```diff
@@ -0,0 +1,256 @@
+
+/* KEEP admin_profiles INTACT */
+
+TRUNCATE TABLE api_request_audits;
+TRUNCATE TABLE api_requests;
+TRUNCATE TABLE api_tokens;
+
+UPDATE appropriate_bodies as item
+    SET name=CONCAT('Appropriate Body ', id);
+
+/* KEEP appropriate_body_profiles INTACT */
+
+/* reset all call off contracts to default values */
+UPDATE call_off_contracts
+    SET raw='{"version":"1.0.0"}',
+        uplift_target=1000,
+        uplift_amount=1000,
+        recruitment_target=1000,
+        set_up_fee=100,
+        revised_target=1111,
+        monthly_service_fee=150;
+
+/* KEEP cohorts INTACT */
+/* KEEP cohorts_lead_providers INTACT */
+/* KEEP completion_candidates INTACT */
+/* KEEP core_induction_programmes INTACT */
+
+/* sequentially rename CPD lead providers */
+UPDATE cpd_lead_providers
+    SET name=CONCAT('Lead Provider ', id);
+
+/* cascades to data_stage_school_changes, data_stage_school_links */
+TRUNCATE TABLE data_stage_schools CASCADE;
+
+/* KEEP declaration_states INTACT */
+
+TRUNCATE TABLE deleted_duplicates;
+
+/* KEEP delivery_partner_profiles INTACT */
+
+/* sequentially rename delivery partners */
+UPDATE delivery_partners
+    SET name=CONCAT('Delivery Partner ', id);
+
+/* KEEP district_sparsities INTACT */
+/* KEEP ecf_ineligible_participants INTACT */
+/* KEEP ecf_participant_eligibilities INTACT */
+
+TRUNCATE TABLE ecf_participant_validation_data;
+
+/* cascades to email_associations */
+TRUNCATE TABLE emails CASCADE;
+
+/* KEEP event_logs INTACT */
+/* KEEP feature_selected_objects INTACT */
+/* KEEP features INTACT */
+
+/* reset all payment adjustments to default values */
+UPDATE finance_adjustments
+    SET amount=1000;
+
+TRUNCATE TABLE finance_profiles;
+
+/* KEEP friendly_id_slugs INTACT */
+
+/* KEEP induction_coordinator_profiles INTACT */
+/* KEEP induction_coordinator_profiles_schools INTACT */
+/* KEEP induction_programmes INTACT */
+/* KEEP induction_records INTACT */
+/* KEEP lead_provider_cips INTACT */
+/* KEEP lead_provider_profiles INTACT */
+
+/* rename ECF lead providers to match their CPD name */
+UPDATE lead_providers AS lp
+    SET name=clp.name
+    FROM cpd_lead_providers AS clp
+    WHERE lp.cpd_lead_provider_id = clp.id;
+
+/* sequentially rename local authorities */
+UPDATE local_authorities
+    SET name=CONCAT('Local Authority ', id);
+
+UPDATE local_authority_districts
+    SET code=local_authority_districts_ref.code,
+        name=local_authority_districts_ref.name
+    FROM local_authority_districts_ref
+    WHERE local_authority_districts.code = local_authority_districts_ref.original_code;
+
+/* KEEP milestones INTACT */
+/* KEEP networks INTACT */
+
+TRUNCATE TABLE npq_application_eligibility_imports;
+TRUNCATE TABLE npq_application_exports;
+
+/* need to rewrite these IDs */
+UPDATE npq_applications
+    SET school_ukprn=schools_ref.uk_prn,
+        school_urn=schools_ref.urn,
+        teacher_reference_number=teacher_profiles_ref.trn,
+        date_of_birth=NULL
```

Comment:
There's the `nino` in here we probably want to hide too.

Reply:
good spot, missed that one.